File: problems/twoSum.js
/**
 * @param {number[]} nums
 * @param {number} target
 * @return {number[]}
 */
var twoSum = function(nums, target) {
    const map = new Map()
    for (let [i, num] of nums.entries()) {
        if (map.has(target - num)) return [i,map.get(target - num)]
        map.set(num, i)
    }
};

File: problems/twoSum.cpp
class Solution {
public:
  vector<int> twoSum(vector<int>& nums, int target) {
  vector<int> awnser;
  for (int i = 0; i < nums.size(); i ++){
    for (int j = i + 1; j < nums.size(); j ++){
      if (nums[i] + nums[j] == target){
        awnser.push_back(i);
        awnser.push_back(j);
      }
    }
  }
  return awnser;
  }
};
